style.css

```diff
@@ -109,3 +104,268 @@ ul {
+  margin-right: 16px;
+  border-radius: 16px;
+}
+
+.nxt_wk {
+  display: flex;
+  width: 295px;
+  height: 220px;
+  margin-left: 40px;
+  margin-right: 40px;
+  border-radius: 8px;
+  padding-bottom: 12px;
+}
+
+.worktwo {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 295px;
+  height: 80px;
+  margin-left: 40px;
+  margin-right: 40px;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.third_work_container {
+  display: grid;
+  height: 592px;
+  grid-column: auto;
+  justify-content: center;
+  margin-top: 114px;
+  margin-left: 24px;
+  margin-right: 24px;
+  border-radius: 16px;
+}
+
+.third_wk {
+  display: flex;
+  width: 295px;
+  height: 220px;
+  margin-left: 40px;
+  margin-right: 40px;
+  border-radius: 8px;
+}
+
+.workthree {
+  display: flex;
+  align-items: center;
+  width: 295px;
+  height: 80px;
+  margin-left: 40px;
+  margin-right: 40px;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.forth_work_container {
+  display: grid;
+  height: 592px;
+  grid-column: auto;
+  justify-content: center;
+  margin-top: 114px;
+  margin-left: 24px;
+  margin-right: 24px;
+  border-radius: 16px;
+}
+
+.forth_wk {
+  display: flex;
+  width: 295px;
+  height: 220px;
+  margin-left: 40px;
+  margin-right: 40px;
+  border-radius: 8px;
+  padding-bottom: 12px;
+}
+
+.workfour {
+  display: flex;
+  align-items: center;
+  width: 295px;
+  height: 80px;
+  margin-left: 40px;
+  margin-right: 40px;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Poppins', sans-serif;
+}
+
+.about_section_container {
+  display: flex;
+  flex-direction: column;
+  height: 1000px;
+  border-radius: 16px;
+  margin-left: 14px;
+  margin-right: 14px;
+}
+
+.about_my {
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  line-height: 52px;
+  font-size: 40px;
+  width: 200px;
+  height: 101px;
+  font-family: 'Poppins', sans-serif;
+  margin-top: 114px;
+  font-weight: 700;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.sec_abt {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 24px;
+  margin-right: 24px;
+  font-size: 16px;
+  margin-top: -5px;
+  height: 140px;
+  gap: 24px;
+  padding: 114px, 24px, 114px, 24px;
+  line-height: 24px;
+  font-family: 'Poppins', sans-serif;
+  color: #344563;
+  width: 300px;
+}
+
+.fr_sk {
+  display: flex;
+  align-items: center;
+  width: 237px;
+  font-weight: 500;
+  color: #000;
+  border-left: 36px;
+  border-right: 102px;
+  height: 24px;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.dropim {
+  display: grid;
+  grid-template-rows: repeat(6, 2fr);
+  row-gap: 40px;
+  font-size: 20px;
+  color: #253858;
+  margin-left: 36px;
+  margin-right: 36px;
+  font-weight: 500;
+  font-family: 'Poppins', sans-serif;
+  line-height: 24px;
+}
```

Comment:
Like ask the previous reviewer , please stick to Figma

<img width="382" alt="image" src="https://user-images.githubusercontent.com/7243132/170830663-a25f1155-4e19-4a34-8305-a8bdaba1f8e3.png">

Figma
<img width="340" alt="image" src="https://user-images.githubusercontent.com/7243132/170830673-dc95ba58-4343-4a07-a6a8-fc332a99df0b.png">